### PR TITLE
Polish up readme

### PR DESCRIPTION
**Pull request by charliesantos**
Doc updates to satisfy PRD requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,19 @@
...
-- We’ve targeted this release to “power users”; documentation is sparse and code samples non-existent. These will come in later (public) releases.
-- Although rare in practice, we reserve the right to introduce breaking API changes in future releases before ultimately going stable/GA.
-- We will tear out the embedded example app and replace it with a proper Quickstart in our first public release. As such, we do not plan to invest any time into fixing or enhancing the current example app.
-- Traditional Twilio Support channels and resources are not available to help. 
-- We welcome your feedback and SDK bug reports (note: non-example app bugs) via the [issues](https://github.com/twilio/twilio-voice-react-native/issues) page.
-
 Twilio's Voice React Native SDK allows you to add real-time voice and PSTN calling to your React Native apps.
 
+- [Usage](#usage)
+- [API Docs](https://github.com/twilio/twilio-voice-react-native/blob/1.0.0-beta.1/docs/voice-react-native-sdk.md)
```
Comment (kpchoy) [inline README.md:8]: This api doc link returns 404 for me

Comment (charliesantos) [reply README.md:8]: beta.1 is not released yet. The link will work once we go beta. To test the link, change 1.0.0-beta.1 to a previous release 
https://github.com/twilio/twilio-voice-react-native/blob/1.0.0-preview.1/docs/voice-react-native-sdk.md

**Review by kpchoy — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -41,8 +27,8 @@ yarn add @twilio/voice-react-native-sdk
...
-For more information on the Voice React Native SDK API, refer to the [API Docs](https://github.com/twilio/twilio-voice-react-native/blob/1.0.0-preview.1/docs/voice-react-native-sdk.md) or try running the [example app](example).
+The following example demonstrates how to make and receive calls. You will need to implement your own `getAccessToken()` method for it to work properly. Please see [Access Tokens](#access-tokens) section for more details or check out the [iOS](https://github.com/twilio/voice-quickstart-ios) and [Android](https://github.com/twilio/voice-quickstart-android) quickstart for examples on how to generate the tokens.
+For more information on the Voice React Native SDK API, refer to the [API Docs](https://github.com/twilio/twilio-voice-react-native/blob/1.0.0-preview.1/docs/voice-react-native-sdk.md) or see our [Reference App](https://github.com/twilio/twilio-voice-react-native-app).
```
Comment (mhuynh5757) [inline README.md:31]: This links to `preview.1` but the above link is to `beta.1`.

Comment (charliesantos) [reply README.md]: Updated. Please take another look @mhuynh5757

**Review by mhuynh5757 — APPROVED** [review]

**Pull request merged**